python/ribasim/tests/test_model.py

```diff
@@ -65,24 +67,15 @@ def test_exclude_unset(basic):
     assert d["solver"]["saveat"] == 86400.0
 
 
-@pytest.mark.xfail(reason="Needs implementation")
+# @pytest.mark.xfail(reason="Needs implementation")
 def test_invalid_node_id(basic):
     model = basic
```

Comment:
I think this test doesn't need to build on the basic model. Running `Node(-1, Point(7.0, 7.0))` by itself should be enough to trigger it. Can you try removing that argument, and just putting `Node(-1, Point(7.0, 7.0))` in the `pytest.raises` block?